resources/views/settings/delete-account.blade.php

```diff
@@ -6,26 +6,94 @@
             <li><a href="{{ localized_route('settings.show') }}">{{ __('Settings') }}</a></li>
         </ol>
         <h1>
-            {{ __('Delete account') }}
+            {{ __('Delete your account') }}
         </h1>
     </x-slot>
 
-    <!-- Form Validation Errors -->
-    @include('partials.validation-errors')
+    @if ($user->isOnlyAdministratorOfOrganization() || $user->isOnlyAdministratorOfRegulatedOrganization())
+        <div role="alert">
+            <x-hearth-alert type="warning">
+                {{ __('You cannot delete your account because you are the only administrator of your organization. Please make another member from your organization an administrator first.') }}
+            </x-hearth-alert>
+        </div>
+    @endif
+    <h2>{{ __('Are you sure you want to delete your account?') }}</h2>
+    <p>{{ __('If you delete your account:') }}</p>
+    @if ($user->context === 'individual')
+        <ul>
+            <li>
+                <p>{{ __('you will no longer be able to access information about the [number] engagements you are participating in') }}
```

Comment:
In all these instances, in order to let the translation work start we'll need to add an actual placeholder. So I'd do something like this as a temporary measure— we can fill in a real number later:

```suggestion
                <p>{{ __('you will no longer be able to access information about the :count engagements you are participating in', ['count' => 0]) }}
```